README.md

```diff
@@ -77,14 +77,16 @@ with the help of the `com.ibm.cloud.cloudant` package.
 
 The purpose of this Java SDK is to wrap most of the HTTP request APIs
 provided by Cloudant and supply other functions to ease the usage of Cloudant.
-Moreover, it has limited support for CouchDB as well.
 This SDK should make life easier for programmers to do what’s really important
 for them: develop.
 
 Reasons why you should consider using Cloudant Java SDK in your
 project:
 
 - Supported by IBM Cloudant.
+- Service compatibility besides Cloudant _Classic_ with
```

Comment:
Perhaps we should go with 
* "limited" compatibility
* I think some Couch 2x features are also supported, so I'd remove the 3.x
* I'd expand the TXE as Cloudant on Transaction Engine as the only place I found the TXE abbreviation is https://blog.cloudant.com/2020/07/02/Best-Practice-for-Transaction-Engine.html